dask_sql/context.py

```diff
@@ -587,69 +592,42 @@ def register_model(
+        Sets sql configuration options to the global dask config
+        A configuration option could be used to set the behavior of certain sql operations.
 
-        Eg: `dask.groupby.agg.split_out` can be used to split the output of a groupby agrregation to multiple partitions.
+        Eg: `sql.groupby.split_out` can be used to split the output of a groupby agrregation to multiple partitions.
 
         Args:
-            config_options (:obj:`Tuple[str,val]` or :obj:`Dict[str,val]`): config_option and value to set
-            schema_name (:obj:`str`): Optionally select schema for setting configs
+            config_options (:obj:`Dict[str,val]`): key, value paris of config_option and value to set.
 
         Example:
             .. code-block:: python
 
                 from dask_sql import Context
 
                 c = Context()
-                c.set_config(("dask.groupby.aggregate.split_out", 1))
+                c.set_config({"sql.groupby.split_out", 1})
                 c.set_config(
                     {
-                        "dask.groupby.aggregate.split_out": 2,
-                        "dask.groupby.aggregate.split_every": 4,
+                        "sql.groupby.split_every": 2,
+                        "sql.identifier.case_sensitive": True,
                     }
                 )
-
-        """
-        schema_name = schema_name or self.schema_name
-        self.schema[schema_name].config.set_config(config_options)
-
-    def drop_config(
-        self, config_strs: Union[str, List[str]], schema_name: str = None,
-    ):
         """
-        Drop user set configuration options from schema
-
-        Args:
-            config_strs (:obj:`str` or :obj:`List[str]`): config key or keys to drop
-            schema_name (:obj:`str`): Optionally select schema for dropping configs
-
-        Example:
-            .. code-block:: python
-
-                from dask_sql import Context
-
-                c = Context()
-                c.set_config(
-                    {
-                        "dask.groupby.aggregate.split_out": 2,
-                        "dask.groupby.aggregate.split_every": 4,
-                    }
+        for config_str in list(config_options.keys()):
+            if not config_str.startswith("sql"):
+                warnings.warn(
+                    f"""
+                Passed non sql config {config_str} which will be ignored.
+                To set non dask-sql configs consider using `dask.config.set_config`
+                """
                 )
-                c.drop_config(
-                    [
-                        "dask.groupby.aggregate.split_out",
-                        "dask.groupby.aggregate.split_every",
-                    ]
-                )
-        """
-        schema_name = schema_name or self.schema_name
-        self.schema[schema_name].config.drop_config(config_strs)
+                del config_options[config_str]
+
+        dask_config.set(config_options)
```

Comment:
This is a good point, my initial reaction in that case is to direct users to use Dask's global config system and point to those methods in our docs.
Maintaining a separate dictionary within a sql context and temporarily using that in a context manager to overwrite any global config is certainly doable but might lead to corner cases in situations where users try using global dask config methods in conjunction with dask sql and don't have their context specific changes get propagated in those scenarios.

Reply:
That's fair to me, in that case would we still need this function? From my perspective, it would be simpler to direct users to instead do something like

```python
dask.config.set(...)

c.sql(...)
```

In the docs, though I'm not sure if there's larger value behind the checks here for `sql` specific config options

Reply:
You're right, with this approach we can get rid of this method and any docs that refer to this, as well as a couple of tests that check this.

I'll also update the config doctstring to point to Dask's set_config method incase users want to use that